README.md

```diff
@@ -60,4 +57,92 @@ directory:
 └── remm
     ├── ReMM.v{{REMM-VERSION}}.hg19.tsv.gz
     └── ReMM.v{{REMM-VERSION}}.hg38.tsv.gz
+```
+
+
+The overall structure of the input directory may look something like this (omitting some files for clarity):
```

Comment:
Does this need to be exactly like this, or is it dependent on the configuration as to how it is laid out?

Reply:
This would be clearer:

`The overall structure of the input directory should look like this with the cadd, local and remm directories being optional, depending on the exomiser configuration:`